### PR TITLE
Cleaning up test build

### PR DESCRIPTION
**Pull request by aericson**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -1,32 +1,24 @@
...
-deps=
-    django==1.3
-
-[testenv:py27-django13]
-basepython=python2.7
+    py.test
 deps=
-    django==1.3
+    django18: Django==1.8.9
+    django19: Django==1.9.2
```
Comment (zoidbergwill) [inline tox.ini]: Should we make these `django>=1.9.0,<1.10.0` type requirements?

File: tox.ini [hunk 2]
```diff
@@ -1,32 +1,24 @@
...
+    pytest-django
+    pytest-pythonpath
 
-[testenv:py26-django14]
-basepython=python2.6
-deps=
-    django==1.4
+[testenv:py27-pyflakes]
+commands = pyflakes templated_email
```
Comment (zoidbergwill) [inline tox.ini]: What about moving to flake8 with the ignored PEP errors?

**Comment by zoidbergwill** [comment]
Yay for tox. :tada:

**Comment by aericson** [comment]
@andersonresende thanks mate

**Pull request merged**